Bio/PDB/StructureBuilder.py

```diff
@@ -196,15 +195,15 @@ def init_atom(self, name, coord, b_factor, occupancy, altloc, fullname,
                                   % (duplicate_fullname, fullname,
                                      self.line_counter),
                                   PDBConstructionWarning)
-        atom = self.atom = Atom(name, coord, b_factor, occupancy, altloc,
-                                fullname, serial_number, element)
+        self.atom = Atom(name, coord, b_factor, occupancy, altloc,
+                         fullname, serial_number, element)
```

Comment:
@peterjc It doesn't make much difference, as we are initiating self.atom anyway in that 
`atom = self.atom = Atom()`
But I think I changed it by looking at the style used in init_chain() and init_residue() where the self.chain/residue were used...